source/rover.js

```diff
@@ -378,10 +397,13 @@ this.calcRam = function() {
 
 this.blockCountChanged = function( value ) {
     this.calcRam();
-    this.activateSensor( 'forward' );
+    this.activateSensor( 'anomaly' );
     this.activateSensor( 'signal' );
     this.activateSensor( 'heading', this.heading );
+    this.activateSensor( 'collision' );
+    this.activateSensor( 'proximity' );
```

Comment:
I think we talked about this... let's make the proximity sensor and the anomaly sensor be one and the same?